src/pages/index.js

```diff
@@ -130,17 +424,116 @@ profileEditButton.addEventListener("click", () => {
   editProfilePopup.open();
 });
 
-//addNewCardPopup.setEventListeners();
-
 addNewCardButton.addEventListener("click", () => {
+  console.log("addnewcardbuttoned clicked");
   addNewCardPopup.open();
   addNewCardFormValidator.resetValidation();
 });
 
-//ALL THE REST
+profileAvatarButton.addEventListener("click", () => {
+ 
+  updateAvatarFormValidator.resetValidation();
+  const { avatar, name, description} = editUserInfo.getUserAvatarInfo();
+
+  profileAvatarUrlInput.value = avatar;
+  profileTitleInput.value = name;
+  profileDescriptionInput.value = description;
```

Comment:
When you open the avatar modal, you don't need to insert anything. The inputs should be empty

And it doesn't have anything to do with the profile inputs